app/src/Header.jsx

```diff
@@ -70,7 +70,7 @@ function Title(props) {
 function TreeSwitcher(props) {
   const t = props.t;
   const others = ['brands', 'flags', 'operators', 'transit'].filter(d => d !== t);
-  const links = others.map(t => (<li><Link to={`index.html?t=${t}`}>{t}/</Link></li>));
+  const links = others.map(t => (<li key={t}><MemoryRouter><Link to={`index.html?t=${t}`}>{t}/</Link></MemoryRouter></li>));
```

Comment:
I add [MemoryRouter](https://reactrouter.com/web/api/MemoryRouter) because test doesn't pass if not it.
`Invariant failed: You should not use <Link> outside a <Router>`